pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/EventDao.sql.stg

```diff
@@ -175,6 +175,7 @@ select q.queued_event_id,
        q.operator_user_id as operator_id,
        ptp.guid           as participant_guid,
        ptp.hruid          as participant_hruid,
+       op.guid            as operator_guid,
```

Comment:
Since we only need to query out one column, should we do a subquery instead of a join to have a bit better performance?

Reply:
I doubt that subquery would have any better performance in this case

Reply:
In fact, this join is a pretty cheap operation as it is unique key lookup
<img width="1317" alt="image" src="https://user-images.githubusercontent.com/71830706/97342220-0c466380-1897-11eb-9b4e-8fc3f3fad7c4.png">